### PR TITLE
deletes test stuff

### PR DESCRIPTION
**Pull request by jkranking**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,16 +2,11 @@
...
-get '/accounts/:id/users', to: 'accounts#users_index', as: 'accounts_users'
-
-get '/users', to: 'users#index'
-
-get '/users/:id', to: 'users#show', as: 'user'
-
-end
+  resources :accounts, only: [:index, :show] do 
+    resources :users
+  end
```
Comment (jkranking) [inline config/routes.rb:9]: commented out # get '/accounts/:id/users', to: 'accounts#users_index', as: 'accounts_users'

I know that this point towards users#index now but I don't know what to do in that action of the users controller
.  do some type of if conditional?

File: app/controllers/accounts_controller.rb [hunk 2]
```diff
@@ -8,8 +8,8 @@ def show
...
   end
 
-  def users_index
-    @users = Account.find(params[:id]).users 
-  end 
+  # def users_index
+  #   @users = Account.find(params[:id]).users 
+  # end 
 
 end
```
Comment (jkranking) [inline app/controllers/accounts_controller.rb:11]: commented this out for now.

File: config/routes.rb [hunk 3]
```diff
@@ -2,16 +2,11 @@
...
-get '/accounts/:id/users', to: 'accounts#users_index', as: 'accounts_users'
-
-get '/users', to: 'users#index'
-
-get '/users/:id', to: 'users#show', as: 'user'
-
-end
+  resources :accounts, only: [:index, :show] do 
+    resources :users, only: [:index, :show]
+  end
```
Comment (jkranking) [inline config/routes.rb:9]: +# get '/accounts/:id/users', to: 'accounts#users_index', as: 'accounts_users'

commented this out as well.  I know that this is now pointing towards the users#index.  But not sure what to do in the index action.  if conditional?

Comment (Andrew-Max) [reply config/routes.rb:9]: This is resolved right?

**Comment by Andrew-Max** [comment]
Will review this later to make sure it all works before merging.

Next step is to get all the new and create actions working. then edit and update.

**Pull request closed without merging**